telegram/_telegramobject.py

```diff
@@ -58,6 +58,12 @@ class TelegramObject:
     The :mod:`pickle` and :func:`~copy.deepcopy` behavior of objects of this type are defined by
     :meth:`__getstate__`, :meth:`__setstate__` and :meth:`__deepcopy__`.
 
+    Tip:
+        Objects of this type can be serialized via Pythons :mod:`pickle` module and pickled
```

Comment:
it's not our module, is `:mod:` still legit in this context?

Reply:
yep